Sources/AudioKit/Nodes/Node+disconnectStrategy.swift

```diff
@@ -0,0 +1,66 @@
+// Copyright AudioKit. All Rights Reserved. Revision History at http://github.com/AudioKit/AudioKit/
+
+import AVFAudio
+
+/// Describes a way to disconnect a node from another node
+public enum DisconnectStrategy {
+    /// Recursively traverse node chain and disconnect
+    /// This strategy will keep connections from downstream nodes
+    /// to nodes that are not in current disconnect path
+    /// For example:
+    /// Mixer1
+    ///          ---> Node1
+    /// Mixer2
+    /// When disconnecting Node1 from Mixer1, Mixer2's connection to Node1
+    /// will be preserved
+    case recursive
+    /// Recursively traverse node chain and deatach nodes
+    /// Audio engine will automatically disconnect detached nodes
+    /// This strategy will not keep any connections from downstream nodes
+    /// For example:
+    /// Mixer1
+    ///          ---> Node1
+    /// Mixer2
+    /// When disconnecting Node1 from Mixer1, Mixer2's connection to Node1
+    /// will not be preserved
+    /// Use only when you are sure that you have only one chain path to node
+    /// that you are disconnecting
+    case deatach
```

Comment:
Spelling should be "detach" everywhere this appears.

Reply:
Done.